packages/slate-types/src/nodes/ImageNode.ts

```diff
@@ -20,8 +20,10 @@ export interface ImageNode extends ElementNode {
     layout: ImageLayout;
     /** matches this regexp: /^\d+(\.\d+)?%$/ */
     width: string;
-    /** matches this regexp: /^\d+(\.\d+)?%$/ */
-    width_factor: string;
+    /**
+     * @deprecated No longer used. Will be dropped in the future.
+     */
+    width_factor: '100%';
```

Comment:
Why do we need to keep the prop in types?

Reply:
I was hesitating to do so, as it would be a BC breaking change. But after thinking this through I couldn't think of any problems caused by dropping it. So let it be. a59fc3cd

Thanks.

Reply:
I have a suspicion that dropping the property caused https://linear.app/prezly/issue/DEV-5142/customers-send-campaign-button-is-greyed-out-and-is-unable-to-send

Because the local state does not match the state coming from backend after a save -- so it keeps believing there are unsaved changes.

Taking care of it now. :)